aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -44,26 +43,82 @@
             expect(Cl.events.calendar.calls.count()).toEqual(5);
         });
 
-        it('Cl.events._handler returns false if direction is not specified', function () {
+        it('has Cl.events._handler to return false if direction is not ' +
+            'specified', function () {
             expect(Cl.events._handler.call(
-                $('.js-trigger')[0],
-                { preventDefault: emptyFunction })
-            ).toEqual(false);
+                $('.js-trigger')[0], this.preventEvent))
+                .toEqual(false);
         });
 
-        it('Cl.events._handler returns undefined if direction is "next"', function () {
+        it('has Cl.events._handler to return undefined if direction is ' +
```

Comment:
should be "returns undefined if direction is specified"
and then you check both assertion in a same test, shouldn't really split them up